README.md

```diff
@@ -28,18 +28,18 @@ Java API which exposes utilities for building update actions and automatic synci
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 ## Usage
 
-commercetools sync is a Java library that could be used to synchronise CTP data in any of the following ways:
+commercetools sync is a Java library that could be used to synchronize CTP data in any of the following ways:
```

Comment:
Is there a reason to switch to American English? :)

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/41eedd2095574945b277efb12f4c047012598531